src/store/index.js

```diff
@@ -29,7 +30,8 @@ export const Store = new Vuex.Store({
     app,
     apps,
     user,
-    config
+    config,
+    links
```

Comment:
This needs to be it's own module due to vuex-persist. Since we are using `modules` in options we can't use reducer to persist only specific states. This would mean the only option of adding it to existing module would be user module which doesn't seem right to me to contain info about private link.